plugins/hls-eval-plugin/src/Ide/Plugin/Eval/Code.hs

```diff
@@ -1,120 +1,120 @@
-{-# LANGUAGE LambdaCase        #-}

```

Comment:
This file is completely altered because of `\r` (it looks like #2597 introducing some `\r`s. C.C. @bradrn).

Reply:
Oh, that’s really bad. The pre-commit hook should *not* change semantics. This is a `stylish-haskell` bug, I think.

Reply:
@bradrn It didn't change the semantics. It just added `\r` on each line, which gives this "huge" git diff. (As my setting auto-removes `\r`s)

Reply:
Thanks for clarifying — I had thought you meant something different by ‘completely altered’.

I’m a bit confused, though. What exactly changed the file? My unrelated PR (merged after this one was submitted) shouldn’t affect this code, unless you merge it… in which case it shouldn’t show up in this commit. Did you run the pre-commit hook by any chance?

Reply:
@bradrn I always rebase my PR even after its submission, so this PR is indeed after your PR.

Reply:
OK, makes sense, thanks!